### PR TITLE
[507] Review expired pages and minor tweaks

### PR DESCRIPTION
**Pull request by saliceti**
## What
List of expired pages: https://ukgovernmentdfe.slack.com/archives/C02GFK8240L/p1643267089007400
Includes minor tweaks like adding TOCs and backlinks
Deleted CIP/PaaS _Current Utilisation No of Applications/services and %_ as it will be hard to keep this accurate.

### CODE REVIEW

**Comment by saliceti** [comment]
[Review tech guidance expired pages](https://trello.com/c/Y9kDWMWJ/507-review-tech-guidance-expired-pages)

File: source/guides/technology-documentation-sites/index.html.md.erb [hunk 1]
```diff
@@ -61,14 +61,14 @@ All the workspace-related settings (workspace ID, API key and API secret) can be
 
 Next we must setup the GitHub Actions to publish:
 
-3. Copy the automation script from [here](https://github.com/DFE-Digital/academy-transfers-technical-documentation/blob/master/source-c4-diagrams/export-private-cloud-diagrams.js) to `<repository-root>/source-c4-diagrams/export-private-cloud-diagrams.js`. 
+3. Copy the automation script from [here](https://github.com/DFE-Digital/academy-transfers-technical-documentation/blob/master/source-c4-diagrams/export-private-cloud-diagrams.js) to `<repository-root>/source-c4-diagrams/export-private-cloud-diagrams.js`.
```
Comment (jonodrew) [inline source/guides/technology-documentation-sites/index.html.md.erb:64]: Can we make the link more descriptive here? It's not great for people using screen readers. See [the guidance on GOV.UK](https://www.gov.uk/guidance/content-design/links) under "Writing link text"

**Review by jonodrew — CHANGES_REQUESTED** [review]
One small comment, and I'm happy to tweak myself if you're busy @saliceti

File: source/guides/technology-documentation-sites/index.html.md.erb [hunk 2]
```diff
@@ -80,10 +80,10 @@ Next we must setup the GitHub Actions to publish:
...
+    Creates a directory for the diagrams to land in (`screenshots`), then utilizes the `structurizr/cli` image to upload the `./source-c4-diagrams/workspace.dsl` file in to the previously created workspace identified by ID `STRUCTURIZR_WORKSPACE_ID`.
 
     A puppeteer automation script is used to automate a headless browser to log in to the cloud service and export the diagrams.
 
-    Finally, the exported files are copied in to their correct positions in the check-out repository file system - ready for the Middleman build process. 
+    Finally, the exported files are copied in to their correct positions in the check-out repository file system - ready for the Middleman build process.
 
 4. Commit, push and then run a build. Make a note of the generated file names (workspace ID will be redacted with asterix) and make any nessicary amends to the GitHub Actions `cp` lines to copy the files defined in your C4 workspace to the correct place in the tech docs directory structure you have defined for your site.
```
Comment (merrilmathew) [inline source/guides/technology-documentation-sites/index.html.md.erb:89]: **nessicary** <- Typo

Comment (saliceti) [reply source/guides/technology-documentation-sites/index.html.md.erb]: Thanks!

File: source/guides/technology-documentation-sites/index.html.md.erb (hunk 1 above)
Comment (saliceti) [reply source/guides/technology-documentation-sites/index.html.md.erb]: Thanks!

**Review by jonodrew — APPROVED** [review]

**Comment by jonodrew** [comment]
Thanks for making those tweaks and reviewing. Looks good to me

**Pull request merged**